src/OpenLoco/src/Input.h

```diff
@@ -36,42 +37,49 @@ namespace OpenLoco::Input
         giant = 1,
     };
 
-    namespace Flags
+    enum class Flags : uint32_t
     {
-        constexpr uint32_t widgetPressed = 1 << 0;
-        constexpr uint32_t flag1 = 1 << 1;
-        constexpr uint32_t flag2 = 1 << 2;
-        constexpr uint32_t toolActive = 1 << 3;
-        constexpr uint32_t flag4 = 1 << 4;
-        constexpr uint32_t flag5 = 1 << 5;
-        constexpr uint32_t flag6 = 1 << 6;
-        constexpr uint32_t viewportScrolling = 1 << 7;
-    }
-
-    namespace MapSelectionFlags
+        none = 0U,
+        widgetPressed = 1 << 0,
+        flag1 = 1 << 1,
+        flag2 = 1 << 2,
+        toolActive = 1 << 3,
+        flag4 = 1 << 4,
+        flag5 = 1 << 5,
+        flag6 = 1 << 6,
+        viewportScrolling = 1 << 7,
+    };
+    OPENLOCO_ENABLE_ENUM_OPERATORS(Flags);
+
+    enum class MapSelectionFlags : uint8_t
     {
-        constexpr uint8_t enable = 1 << 0;
-        constexpr uint8_t enableConstruct = (1 << 1);
-        constexpr uint8_t enableConstructionArrow = 1 << 2;
-        constexpr uint8_t unk_03 = 1 << 3;
-        constexpr uint8_t unk_04 = 1 << 4; // Vehicle orders?
-        constexpr uint8_t catchmentArea = 1 << 5;
-        constexpr uint8_t hoveringOverStation = 1 << 6;
+        none = 0U,
+        enable = 1 << 0,
+        enableConstruct = (1 << 1),
+        enableConstructionArrow = 1 << 2,
+        unk_03 = 1 << 3,
+        unk_04 = 1 << 4, // Vehicle orders?
+        catchmentArea = 1 << 5,
+        hoveringOverStation = 1 << 6,
     };
+    OPENLOCO_ENABLE_ENUM_OPERATORS(MapSelectionFlags);
 
-    namespace KeyModifier
+    enum class KeyModifier : uint8_t
     {
-        constexpr uint8_t shift = 1 << 0;
-        constexpr uint8_t control = 1 << 1;
-        constexpr uint8_t unknown = 1 << 2;
-        constexpr uint8_t cheat = 1 << 7;
+        none = 0U,
+        shift = 1 << 0,
+        control = 1 << 1,
+        unknown = 1 << 2,
+        cheat = 1 << 7,
+        full = 0xFF,
```

Comment:
I think `invalid` would be a better name and add a comment next to it saying default value for keymodifier.